inference-engine/src/transformations/include/transformations/common_optimizations/mul_conv_fusion.hpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <functional>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API MultiplyConvolutionFusion;
+class TRANSFORMATIONS_API MultiplyGroupConvolutionFusion;
+class TRANSFORMATIONS_API MultiplyConvolutionBackpropDataFusion;
+class TRANSFORMATIONS_API MultiplyGroupConvolutionBackpropDataFusion;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief Multiply->Convolution fusion replaces following graph:
+ *
+ *   +-------+   +----------+
+ *   | Input |   | Constant |
+ *   +-------+   +----------+
+ *       |            |
+ *       ------  ------
+ *            |  |
+ *            v  v
+ *         +----------+            +---------+
+ *         | Multiply |            | Weights |
+ *         +----------+            +---------+
+ *              |                       |
+ *              -----------    ----------
+ *                        |    |
+ *                        v    v
+ *                   +----------------+
+ *                   | Convolution Op |
+ *                   +----------------+
+ *
+ * to following:
+ *
+ *                           +---------+   +----------+
+ *                           | Weights |   | Constant |
+ *                           +---------+   +----------+
+ *                                |            |
+ *                                ------  ------
+ *                                     |  |
+ *                                     v  v
+ *          +-------+              +----------+
+ *          | Input |              | Multiply |
+ *          +-------+              +----------+
+ *              |                       |
+ *              -----------    ----------
+ *                        |    |
+ *                        v    v
+ *                   +----------------+
+ *                   | Convolution Op |
+ *                   +----------------+
+ *
+ * where 'Convolution Op' is one of:
+ * - Convolution
+ * - ConvolutionBackpropData
+ * - GroupConvolution
+ * - GroupConvolutionBackpropData
+ *
+ * Restrictions:
+ * - weights' shape is static
+ * - the first dimension of constant input to Multiply has to be 1
```

Comment:
I think this is the wrong statement just because Constant may have (C,1,1) shape which is correct.